ApoapsisFunctionHW3.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jun 12 15:16:48 2024
+
+@author: wills
+"""
+
+import numpy as np
+
+
+def visviva(mu,r,a):
+    '''
+    Calculate the velocity at point in orbit.
+
+    Parameters
+    ----------
+    mu : int
+        Gravitational Constant (km^2/s^2).
+    r : int
+        Distance from center body (km).
+    a : int
+        semi-major axis (km).
+
+    Returns
+    -------
+    v : float
+        velocity at specifies orbit point.
+    '''
+    
+    v = np.sqrt(mu*((2/r)-(1/a)))
+    
+    return v
+
+
+
+
+def delVreq(ApogeeChange):
+    '''
+    Calculate the difference in velocities at given points in orbit.
+    
+    Parameters
+    ----------
+    mu : int
+        Gravitational Constant (km^2/s^2).
+    rF : int
+        Final distance from center body (km).
+    aF : int
+        Final semi-major axis (km).
+    rI : int
+        Initial distance from center body (km).
+    aI : int
+        Initial semi-major axis (km).
+
+    Returns
+    -------
+    v : float
+        dV necessary to change from specified apogees.
+    '''
+    mu = 398600
+    rF = 6563
+    rI = 6563
+
+    aI = ((6378+185)+(6378+185))/2

```

Comment:
Right now you have it hardcoded to find the dV from a 185 km circular orbit to the desired orbit. 

Instead, line 63 should be `aI = ((6378+185)+(6378+desiredApogee))/2 ` and line 64 should be `aF = ((6378+185)+(6378+410000))/2`

That reflects that the input is the desired apogee and then we're calculating the delta-v needed to get to the 410,000 km apogee orbit

(Sorry if I didn't catch this in your initial code drop for HW2)